### DISCUSSION TITLE
Smoke kit questions

### DISCUSSION THREAD

**Discussion by MrPeens** [gpstar81/GPStar-proton-pack#258] [Q&A] [answered]
I'm about to install the recommended Wasteland smoke kit into my GPstar pack. I'm using the larger 6000 tallentcell battery and will only have N-filter smoke. I can't find exact specs on how many amps the smoke pump/heating coil or the full GPstar kit (pack and wand with inner cyclotron add ins) will draw. 

Question 1: can I split the 5v usb out from the tallentcell to run both smoke pump and GPstar boards instead of using a buck converter from the 12v battery out for smoke?

Question 2: is it a "better" smoke effect to have the fan and LED triggered directly from the GPstar board instead of from the little circuit board that comes with the Wasteland kit?  I assume there is a time delay with the GPstar board whereas the Wasteland board just turns everything (pump/coil, fan and led) on as soon as smoke is called for?

Thanks for the help and I apologize if this has been answered on facebook as I do not have an account.

**Comment by DustinGrau** [comment, marked as answer]
1) You will want to use the buck converter because you're going to draw way more power direct from the battery. The 5V USB port tops out at 2A and the system can already draw a little over 1A when all the devices are running/firing. The smoke alone will pull over 1A putting you at the limit or over. The 12V output can support up to 3A output by comparison.

2) Yes, you can safely power the LED and fan (blower) from the GPStar controller and that is preferred due to the more precise control over the timing on those devices. The light, fan, and smoke are naturally triggered at different times to make the effects more pronounced (read: running the smoke generator a 1-2 seconds before the fan kicks in to blow it out).
